lib_board_support/api/boards/xk_eth_xu316_dual_100m/board.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#ifndef __XK_ETH_XU316_DUAL_100M_BOARD_H__
+#define __XK_ETH_XU316_DUAL_100M_BOARD_H__
+
+#include <boards_utils.h>
+#if (BOARD_SUPPORT_BOARD == XK_ETH_XU316_DUAL_100M) || defined(__DOXYGEN__)
+#include <xccompat.h>
+#include "smi.h"
+
+#ifdef __XC__
+#define NULLABLE_CLIENT_INTERFACE(tag, name) client interface tag ?name
+#else
+#define NULLABLE_CLIENT_INTERFACE(type, name) unsigned name
+#endif
+
+
+/**
+ * \addtogroup xk_eth_xu316_dual_100m
+ *
+ * API for the xk_eth_xu316_dual_100m board.
+ * @{
+ */
+
+/** Task that connects to the SMI master and MAC to configure the
+ * DP83826E PHYs and monitor the link status. Note this task is combinable
+ * (typically with SMI) and therefore does not need to take a whole thread.
+ * 
+ * Note it may be necessary to modify R3 and R23 according to which
```

Comment:
I don't understand this comment. What are R3 and R23?

Reply:
On the base board - zero ohm links